### ISSUE TITLE
pypy3 builds disabled, due to pypy3 pylint issue

### ISSUE THREAD

**Issue by KennethNielsen** [SoCo/SoCo#499]
When travis calls pylint with pypy3 it fails with this traceback:

```python
Traceback (most recent call last):
  File "/home/travis/virtualenv/pypy3-2.4.0/bin/pylint", line 11, in <module>
    sys.exit(run_pylint())
  File "/home/travis/virtualenv/pypy3-2.4.0/site-packages/pylint/__init__.py", line 13, in run_pylint
    Run(sys.argv[1:])
  File "/home/travis/virtualenv/pypy3-2.4.0/site-packages/pylint/lint.py", line 1300, in __init__
    linter.check(args)
  File "/home/travis/virtualenv/pypy3-2.4.0/site-packages/pylint/lint.py", line 726, in check
    self._do_check(files_or_modules)
  File "/home/travis/virtualenv/pypy3-2.4.0/site-packages/pylint/lint.py", line 855, in _do_check
    self.check_astroid_module(ast_node, walker, rawcheckers, tokencheckers)
  File "/home/travis/virtualenv/pypy3-2.4.0/site-packages/pylint/lint.py", line 932, in check_astroid_module
    checker.process_tokens(tokens)
  File "/home/travis/virtualenv/pypy3-2.4.0/site-packages/pylint/checkers/refactoring.py", line 214, in process_tokens
    elif six.PY3 and token.exact_type == tokenize.COMMA:
AttributeError: 'TokenInfo' object has no attribute 'exact_type'
```

Will need more digging into.

The pypy3 builds have been disabled for new.

**Comment by KennethNielsen** [comment]
If anyone feels up to it, I would welcome help on this one. I can think of quite a few different possibilities that could produce an error like this and they should be investigated.

 * The version of pypy3 that runs on travis may be an older one, that tracks and older version of Python3. I think I saw somewhere that it was pypy3 2.4 that tracks Python 3.2. Python 3.2 is not longer supported by pylint, so it may be, that this would have failed on normal Python 3.2 as well. In that case, if there is a newer pypy3 we should as travis to update, or find whetever issue in which they are most likely tracking a similar request.
 * It could be a bug in pypy3. Something they simply do not support properly. If that turns out to be likely, then it should be reported and linked to here.
 * For the reasons in the first point, I think it is unlikely that it is a bug in pylint. Then it shouldn't pass on all the regular Pythons since 3.3.

Basically, it is somewhat of an investigative task.

**Comment by ghcs27** [comment]
From the traceback one can see that pylint is missing the TokenInfo.exact_type attribute which was introduced in python 3.3.

The python version in the pypy3 builds is (see Travis logs)
```
$ python --version

Python 3.2.5 (b2091e973da6, Oct 19 2014, 18:29:55)
[PyPy 2.4.0 with GCC 4.6.3]
```
so this is probably the reason for the error.
Newer [versions](http://doc.pypy.org/en/latest/index-of-release-notes.html) of PyPy already support python up to version 3.5 - I'd just try if one of them works.

**Comment by KennethNielsen** [comment]
This issue seems to have been fixed. Closing.
